public/doc/openapi-3-v3.0.json

```diff
@@ -3292,6 +3292,11 @@
             "title": "MqttClientId",
             "type": "string"
           },
+          "enabled": {
+            "type": "boolean",
+            "title": "Enabled",
+            "default": false
```

Comment:
@hsong-rh You also need readOnly its missing for the other attributes too. info, mqtt_client_id, enabled